src/main/java/io/jenkins/plugins/artifact_manager_jclouds/s3/S3BlobStore.java

```diff
@@ -262,6 +262,7 @@ public String toString() {
         sb.append(", region='").append(getRegion()).append('\'');
         sb.append(", deleteArtifacts='").append(isDeleteArtifacts()).append('\'');
         sb.append(", deleteStashes='").append(isDeleteStashes()).append('\'');
+        sb.append(", useTransferAcceleration='").append(getConfiguration().getUseTransferAcceleration()).append('\'');
```

Comment:
The new field is not even used in this class.
```suggestion
```

Reply:
Removed

Reply:
If you use this feature with an S3 bucket fer acceleration the "Validate S3 Bucket Configuration" button on the plugin config page gives this error
> GetBucketLocation failed
> com.amazonaws.services.s3.model.AmazonS3Exception: S3 Transfer Acceleration is disabled on this bucket